src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -421,6 +421,9 @@ export default {
 		},
 		hasOnePageOfLoans() {
 			return this.totalCount <= this.limit;
+		},
+		showHelpMeChooseFeat() {
+			return this.enableHelpmeChoose && this.allLoans.length > 8;
```

Comment:
Should we be using `this.allLoans.length` or the `totalCount` computed prop?

Reply:
Looks like `totalCount` is for all loans in the channel. `this.allLoans.length` is counting current loans in the page.

Logging this, I have a different number for livestoc for ex.
94 with `totalCount`
12 with `this.allLoans.length`

Reply:
Interesting that `totalCount` doesn't appear to be respecting filters. Thanks for investigating!